src/sentry/search/eap/spans/attributes.py

```diff
@@ -420,6 +428,53 @@
     ]
 }
 
+DEPRECATED_ATTRIBUTES: list[dict[str, Any]] = []
+try:
+    with open(os.path.join(SENTRY_CONVENTIONS_DIRECTORY, "deprecated_attributes.json"), "rb") as f:
+        DEPRECATED_ATTRIBUTES = json.loads(f.read())["attributes"]
+except Exception:
+    logger.exception("Failed to load deprecated attributes from 'deprecated_attributes.json'")
```

Comment:
In addition to this check 🤔 would it help to have a test that would effectively make sure that the json is accessible and valid?